packages/web/pages/assets/index.tsx

```diff
@@ -174,7 +198,23 @@ const Assets: NextPage = observer(() => {
         ibcBalances={ibcBalances}
         onDeposit={onTableDeposit}
         onWithdraw={onTableWithdraw}
-        onBuyOsmo={() => transferConfig?.buyOsmo()}
+        onBuyOsmo={() => {
+          transferConfig?.buyOsmo();
+
+          const cryptoBalance = nativeBalances.find(
+            (coin) => coin.balance.denom.toLowerCase() === "OSMO"
```

Comment:
Wouldn't lower case be `osmo` instead of `OSMO`?

Reply:
Nice catch, thanks!